packages/core/src/main/conversation/message/MessageBuilder.ts

```diff
@@ -184,8 +186,9 @@ export class MessageBuilder {
     messageId = MessageBuilder.createId(),
     expectsReadConfirmation?: boolean,
     legalHoldStatus?: LegalHoldStatus,
+    debugOptions?: DebugOptions,
```

Comment:
Now we have many optionals: `expectsReadConfirmation`, `legalHoldStatus` & `debugOptions`

Let's put them all together in one `options` object.